### PR TITLE
build: Release kedro-docker 0.5.0

### PR DESCRIPTION
**Pull request by lrcouto**
## Description

Release `kedro-docker` 0.5.0

Depends on #496

## Checklist

- [x] Opened this PR as a 'Draft Pull Request' if it is work-in-progress
- [x] Updated the documentation to reflect the code changes
- [x] Added a description of this change in the relevant `RELEASE.md` file
- [x] Added tests to cover my changes

### CODE REVIEW

File: kedro-docker/RELEASE.md [hunk 1]
```diff
@@ -6,6 +6,9 @@
 
 ## Breaking changes to the API
 
+# Release 0.5.0
+## Major features and improvements
+* Add compatibility with `kedro` 0.19.x
```
Comment (noklam) [inline kedro-docker/RELEASE.md:11]: Technically I think even the older version works with 0.19. The only changes is we assume 0.19.x template so requirements is moved 1 level up. I will say this is a breaking changes too, and add a note that if people need to use this for old projects, they should either use a older `kedro-docker` version or move the requirements file up.

File: kedro-docker/RELEASE.md [hunk 2]
```diff
@@ -6,6 +6,13 @@
 
 ## Breaking changes to the API
 
+# Release 0.5.0
+## Major features and improvements
+* Add compatibility with `kedro` 0.19
+
+## Breaking changes
+* `kedro-docker` was updated to use the `0.19` version of the Kedro starters. If you need to use `kedro-docker` for old project, you can either use an older version or move the requirements file up.
```
Comment (merelcht) [inline kedro-docker/RELEASE.md:14]:
```suggestion
* `kedro-docker` was updated to use the `0.19` version of the Kedro starters. If you need to use `kedro-docker` for an old project, you can either use an older version or move the requirements file up from `src` to the top-level project directory.
```

**Review by merelcht — APPROVED** [review]
Ship it! :shipit:

**Review by stichbury — APPROVED** [review]
LGTM 🌟

**Pull request merged**